components/actions/actionResponses.js

```diff
@@ -48,6 +48,15 @@ async function learn_something_else(say) {
   return await say(
     `You clicked *Learn About Something Else*\n
     We encourage you to read through our Community Handbook: https://handbook.chaoss.community/, and if you still can't find what you're looking for, feel free to ask your question in our #newcomers slack channel.`
+
+  );
+}
+
+async function faqs(say) {
+  return await say(
+    `You clicked *FAQs*\n
+    We encourage you to read through our Frequently Asked Questions: https://handbook.chaoss.community/community-handbook/about/general-faq, and if you still can't find what you're looking for, feel free to ask your question in our #newcomers slack channel.`
```

Comment:
Hi! I missed this the last time. Please make "#newcomers" a link to the newcomers channel on Slack. You can do that by replacing `#newcomers` with `<#C0207C3RETX>`. 

Ps: C0207C3RETX is the channel ID for the newcomers channel in the CHAOSS Slack workspace.